webpack.config.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+/* eslint-env node */
+
+const path = require('path');
+const CleanWebpackPlugin = require('clean-webpack-plugin');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+
+module.exports = {
+  devtool: 'source-map',
+
+  entry: [
+    path.resolve(__dirname, 'src', 'sidebar', 'app', 'app.js'),
+  ],
+
+  output: {
+    // build to the extension src vendor directory
+    path: path.resolve(__dirname, 'build'),
+    filename: path.join('sidebar', 'app.js'),
+  },
+
+  plugins: [
+    // cleanup build folder
+    new CleanWebpackPlugin([
+      'build'
+    ]),
+    // Moves files
+    new CopyWebpackPlugin([
+      {from: path.join('src')}
+    ], {
+      ignore: [
+          '.DS_Store',
```

Comment:
this is strange

Reply:
Can't remember why it's here. Has been removed.